services/src/main/resources/META-INF/cxs/events/common.json

```diff
@@ -0,0 +1,34 @@
+{
+  "propertyTypes": [
+    {
+      "itemId": "id",
+      "type": "id"
+    },
+    {
+      "itemId": "cdp_source"
```

Comment:
Should we really have the cdp fields here ? I thought we would have the Event.class fields here and then create a layer to translate them to cdp in the GraphQL code which would seem more coherent.

Reply:
Removed common.json completely